src/System.Data.Odbc/src/System/Data/Odbc/OdbcParameter.cs

```diff
@@ -346,7 +346,7 @@ private int GetColumnSize(object value, int offset, int ordinal)
             {
                 if (ODBC32.SQL_C.NUMERIC == _typemap._sql_c)
                 {
-                    cch = 62;  // (DecimalMaxPrecision+sign+terminator)*BytesPerUnicodeCharater
+                    cch = 62;  // (DecimalMaxPrecision+sign+terminator)*BytesPerUnicodeCharacter
```

Comment:
It's a comment in C# style explaining how the constant was arrived at, rather than a real usage site.